cantor-archive/src/main/java/com/salesforce/cantor/archive/FileEventsArchiver.java

```diff
@@ -24,12 +25,27 @@
 
 import static com.salesforce.cantor.common.CommonPreconditions.checkArgument;
 
-public class EventsArchiver extends AbstractBaseArchiver {
-    private static final Logger logger = LoggerFactory.getLogger(EventsArchiver.class);
+public class FileEventsArchiver extends AbstractBaseArchiver implements EventsArchiver<Path> {
```

Comment:
@AlexanderKouthoofd `s/FileEventsArchiver/EventsArchiverOnFile/` and the same for the rest, so it's consistent with other `OnSomething` class names.

Reply:
I renamed everything in the package to be `...OnFile`